scripts/cipublish

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -ex
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Create zip file for release.
+"
+}
+
+if [ "${BASH_SOURCE}[0]}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+
+    git rev-parse HEAD > version.txt
+    zip -r dist/pwd-river-dispatches-release.zip usgs-details.html version.txt \
+    images/ CHANGELOG.md media/ edge_includes/ Dispatches_v04t* -x *.zip
```

Comment:
Can we indent this line one level deeper to communicate easier that it is an extension of the previous one?